torch/tensors/uint1_7.py

```diff
@@ -0,0 +1,215 @@
+import torch
```

Comment:
1_7 is a terrible name. How about `sub_byte_uint` or `tiny_uint`

Reply:
OK, I can use sub_byte_uint